index.html

```diff
@@ -2316,11 +2316,39 @@ <h2>Privacy Considerations</h2>
       <section>
         <h3>Unlinkability</h3>
 
-        <p class="issue">
-When the contents of a digitally signed payload contains correlatable
-identifiers, those identifiers can be used to track individuals. A static
-digital signature is a correlatable identifier. There are digital signature
-schemes that provide uncorrelatable digital signatures.
+        <p>
+When the contents of a digitally signed payload contains a correlatable
+identifier, that identifier can be used for tracking purposes by a verifier,
+which might lead to a violation of an expected level of privacy. An example of
+such an identifier is a shopping loyalty card number. However, some of these
+identifiers might not be immediately apparent, such as a static digital
+signature or a cryptographic hash of an image, which can also be used as a
+correlatable identifier.
+        </p>
+
+        <p>
+It is possible to create a digitally signed payload that does not have any
+correlatable tracking identifiers while also providing some level of assurance
+that the payload is trustworthy for a given interaction. This characteristic is
+called <dfn>unlinkability</dfn> which ensures that no correlatable identifiers
+are used in a digitally signed payload while providing some level of trust.
+        </p>
+
+        <p>
+There are at least two aspects to providing unlinkability. The first aspect is
+to 1) ensure that every data field used in the message is never repeated in a
+future message, or 2) ensure that any repeated field(s) provide adequate herd
```

Comment:
I can't tell whether the `at least two aspects` is meant to refer to the `1)` and `2)`, or to `the first` and an unstated `second`.  That knowledge is necessary to rephrase this paragraph, which rephrasing is currently necessary.

Reply:
Reworded in 4994c5b.